framework/rendering/hpp_render_target.h

```diff
@@ -19,6 +19,7 @@
 
 #include "core/hpp_image.h"
 #include <functional>
+#include <memory>
```

Comment:
Any reason for adding this? If this is required (by the sample) you'll also have to update the copyright in this header to 2023 ;)

Reply:
Yes, `unique_ptr` needs it. The other samples seem to have gotten lucky with it being included through some other header.

>  you'll also have to update the copyright in this header to 2023

Done (weird, but ok)